raionart_kaetzlein_bloodline/custom_wearables/raionart_kaetzlein_cap.json

```
@@ -0,0 +1,6 @@
+{
+    "reference_name": "raionart_kaetzlein_cap",
+    "display_name": "Raionart's Kätzlein Cap",
```

```suggestion
    "display_name": "Raionart's Kätzlein cap",
```

Just for consistency